### ISSUE TITLE
Render custom components inside markdown content

### ISSUE THREAD

**Issue by phaistonian** [storybooks/marksy#62]
Hello,

Is something like this doable?

```
- list item 
- list item 2
- list item containing <Custom>custom component</Custom>
```

When `<Custom>custom component</Custom>` is not inside the list it works fine, when it is, then the component is escaped.

Ideas?

**Comment by ChristophLabacher** [comment]
Unfortunately the same seems true, even when the custom component is just contained within a simple Markdown paragraph. 

So even this does not work:
```
I need an <Custom>inline</Custom> component.
```

I too would love to know whether there is any way to get custom inline components to work?

**Comment by tscheiki** [comment]
+1

**Comment by xtrinch** [comment]
+256, this is pretty much a deal breaker

**Comment by christianalfoni** [comment]
Yeah, this is just the behaviour of the parser. I think maybe even doing:

```js
<div>I need a <Custom>inline</Custom> component</div>
```

Would work
